tests/functionality/index.js

```diff
@@ -1,15 +1,15 @@
-const { settingsScreen } = require('./settingsScreen.tests');
-const { homeScreen } = require('./homeScreen.tests');
-const { carbEntryScreen } = require('./carbEntryScreen.tests');
-const { pumpSimulatorScreen } = require('./pumpSimulatorScreen.tests');
-const { cgmSimulatorScreen } = require('./cgmSimulatorScreen.tests');
-const { deliveryLimitsScreen } = require('./deliveryLimitsScreen.tests');
+const settingsScreen = require('./settingsScreen.tests');
```

Comment:
[nit] Consider adding `Tests` suffix on all of these so it is clear you aren't importing the settings screen, but rather the settings screen tests.